docs/data.md

```diff
@@ -155,9 +155,7 @@ current mapset.
 ## Using local scratch space for temporary data
 
 ### Running a single job
-If you are processing a large number of temporary files with GRASS GIS (e.g., temporary rasters or mapsets), it can be useful to write the   
-temporary files to local scratch space on the compute node (can decrease run time). In your
-[job submission script](jobs.md#running-a-single-job-calling-a-grass-module) you will need to add the following code, e.g.:  
+If you are processing a large number of temporary files with GRASS GIS (e.g., temporary rasters or mapsets), it can be useful to write the temporary files to local scratch space on the compute node (can decrease run time). In your [job submission script](jobs.md#running-a-single-job-calling-a-grass-module) you will need to add the following code, e.g.:  
```

Comment:
This and couple other paragraphs are re-flown (now as a single line which is fine with me as long as the linter is happy about it) but they still have trailing spaces after the last sentence (the linter accepts it because it means hard line break in Markdown, but that's not what we are trying to do here at the end of the paragraph).

Reply:
No worries! I can add the JOBID and TMPDIR to the list of known words, do I need to create another PR to do that? 

Sorry about re-formatting the paragraphs -- I will remove trailing spaces once you answer the above question!

Reply:
> I can add the JOBID and TMPDIR to the list of known words, do I need to create another PR to do that?

This PR is the right one to do that since you are introducing the words here...and the configuration and linting will just fall in place as expected.

Reply:
Okay, I think I figured out how to do it now!

Reply:
Great. It runs now. I'll wait for #45 before merging this to remove the other issues.